coding-concepts.js

```diff
@@ -1,51 +1,64 @@
-// ASSESSMENT 1: Coding conceptual questions
+// // ASSESSMENT 1: Coding conceptual questions
 
-// Examine the following examples.
+// // Examine the following examples.
 
-// First, predict the outcome based on your understanding of the code.
-// Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
+// // First, predict the outcome based on your understanding of the code.
+// // Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
 
-// --------------------1) What will this log?
+// // --------------------1) What will this log?
 
-var cohort = "Charlie 2021"
+// var cohort = "Charlie 2021"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer: 12
+// // b) Verify and explain: The .length method returns the number of characters in the string
 
 
-// --------------------2) What will this log?
+// // --------------------2) What will this log?
 
-var greeting = "Hello World!"
+// var greeting = "Hello World!"
 // console.log(greeting[3])
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer: l
+// // b) Verify and explain: The number in [] is the index being requested and the character located at index 3 is l.
 
 
-// --------------------3) What will this log?
+// // --------------------3) What will this log?
 
-var languages = ["JavaScript", "Ruby", "Python", "C++"]
-var index = 1
+// var languages = ["JavaScript", "Ruby", "Python", "C++"]
+// var index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer: Ruby
+// // b) Verify and explain: If I'm being honest I took a bit of an educated guess here because I was a little thrown off by 
+//having a variable called index but if the goal was to find the value held at index 1, it would be Ruby because 
+//the index of an array starts at 0, so Javascript = 1 and Ruby = 2.
```

Comment:
yeah, correct "Javascript" being 1st is index = 0
and "Ruby" being 2nd is index = 1